src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TeamsController.cs

```diff
@@ -104,29 +104,26 @@ public JsonResult Create(TeamViewModel teamViewModel)
                     {
                         var newAddedCaptain = _playerService.Create(teamViewModel.Captain.ToCreatePlayerDto());
                         teamViewModel.Captain.Id = newAddedCaptain.Id;
+
                     }
 
+                    teamViewModel.Roster = new List<PlayerNameViewModel> {
+                        new PlayerNameViewModel {
+                            Id = teamViewModel.Captain.Id,
+                            FirstName = teamViewModel.Captain.FirstName,
+                            LastName = teamViewModel.Captain.LastName
+                        }
+                    };
+
+                    var playersIdToAddToTeam = SeparatePlayers(teamViewModel);
+
                     var team = teamViewModel.ToCreateTeamDto();
                     var createdTeam = _teamService.Create(team);
                     teamViewModel.Id = createdTeam.Id;
 
-                    createdTeam.SetCaptain(new PlayerId(createdTeam.Captain.Id));
                     _playerService.AssingPlayerToTeam(teamViewModel.Captain.ToDomain(), createdTeam.Id);
```

Comment:
Why do we need this one?

Reply:
Actualy we don't need it, as in _teamService.AddPlayers we add player to roster